### DISCUSSION TITLE
Could you make action to Get Table of Contents of PDF file like one in Zotfile attachment manager?

### DISCUSSION THREAD

**Discussion by youngchiron** [windingwind/zotero-actions-tags#240] [Q&A]
Could you make action to Get Table of Contents of PDF file like one in Zotfile attachment manager? 
I couldn't file one with the same function.
Thanks in advance.

**Comment by windingwind** [comment]
Could you please provide me with a link or description related to what you are looking for?

**Comment by youngchiron** [comment]
Thanks. 
For example, attached file's zotero link is zotero://select/library/items/CDY95KVY, its table are extracted in a note are....

- [[A Companion to Joachim of Fiore](zotero://open-pdf/0_CDY95KVY/4)](zotero://open-pdf/0_CDY95KVY/4)   
- [[Copyright](zotero://open-pdf/0_CDY95KVY/5)](zotero://open-pdf/0_CDY95KVY/5)    
- [[Contents](zotero://open-pdf/0_CDY95KVY/6)](zotero://open-pdf/0_CDY95KVY/6)    
- [[Acknowledgements](zotero://open-pdf/0_CDY95KVY/8)](zotero://open-pdf/0_CDY95KVY/8)    
- [[Notes on Contributors](zotero://open-pdf/0_CDY95KVY/9)](zotero://open-pdf/0_CDY95KVY/9)    
- [[Introduction: Joachim of Fiore in the History of Western Culture](zotero://open-pdf/0_CDY95KVY/12)](zotero://open-pdf/0_CDY95KVY/12)    
- [[1 The Life and Works of Joachim of Fiore—An Overview](zotero://open-pdf/0_CDY95KVY/31)](zotero://open-pdf/0_CDY95KVY/31)    
- [2 Joachim the Theologian: Trinitarian Speculation and](zotero://open-pdf/0_CDY95KVY/52)


and it looks like...

![image](https://github.com/windingwind/zotero-actions-tags/assets/150308357/4daef924-d442-4e29-9a5d-bcfa3b372b78)

**Reply by windingwind** [reply]
Please do not use the attachment note, which is going to be deprecated

**Reply by youngchiron** [reply]
I don't understand what you mean by "the attachment note, which is going to be deprecated". 
You mean.. Don't use a note attached to an attached file in Zotero? 
Or you mean, Use replay function in this board?

**Reply by windingwind** [reply]
> Don't use a note attached to an attached file in Zotero?

yes

**Reply by zyx335588** [reply]
if the TOC was put in a nornal child note, could it be implemented?

**Reply by windingwind** [reply]
you may try, though I'm not sure why would you need a TCO in a note...

**Reply by zyx335588** [reply]
I can use TOC and/or abstract (if a paper has) to quickly find the paper or paper section I needed without opening every paper. Althogh the abstract also exist at the sidebar, it seems not appropriate for reading.

**Reply by windingwind** [reply]
yeah. i mean, you could try, but sorry i won't be able to help with that.
